projects/ngrx.io/content/guide/signals/signal-store/index.md

```diff
@@ -141,6 +141,8 @@ export class BooksComponent {
 
 </code-example>
 
+The value wrapped inside a Signal object can be easily accessed at any time by calling the Signal as a function with no arguments.
```

Comment:
Thanks for the PR @goodnic 

I think when this isn't clear that this text isn't that helpful, it probably means the user is new to signals. Instead it would be better to point to the signals docs in that case.

This can also be applicable to other packages of NgRx, but this isn't something that we do.
That's why I'm not a big fan of this PR, and I would rather leave it as is.